bin/run.sh

```diff
@@ -20,6 +20,12 @@ base_dir=$(pwd)
 solution_dir=$(realpath $2)
 output_dir=$(realpath $3)
 
+# Copy solution to /tmp
+mkdir -p /tmp/solution
```

Comment:
would suggest using `mktemp -f` which will generate you a unique directory.

Reply:
Neat, I didn't know about that.
I used it, but had to do some modifications to the other scripts since the output is not deterministic anymore (which is probably a good thing).